couchbase/transactions/transactions_config.hxx

```diff
@@ -110,54 +116,99 @@ class transactions_config
      * @brief Set the expiration time for transactions.
      *
      * @param duration desired expiration for transactions. see @expiration_time().
+     * @return reference to this, so calls can be chained.
      */
     template<typename T>
-    void expiration_time(T duration)
+    transactions_config& expiration_time(T duration)
     {
         expiration_time_ = std::chrono::duration_cast<std::chrono::nanoseconds>(duration);
+        return *this;
     }
 
-    void metadata_collection(const couchbase::transactions::transaction_keyspace& keyspace)
+    /**
+     * Set the transaction's metadata collection.
+     *
+     * @param keyspace The collection to use for the transaction metadata.
+     * @return reference to this, so calls can be chained.
+     */
+    transactions_config& metadata_collection(const couchbase::transactions::transaction_keyspace& keyspace)
     {
         metadata_collection_ = keyspace;
+        return *this;
     }
 
-    void metadata_collection(const std::string& bucket, const std::string& scope, const std::string& collection)
+    /*void metadata_collection(const std::string& bucket, const std::string& scope, const std::string& collection)
     {
         metadata_collection_.emplace(bucket, scope, collection);
-    }
+    }*/
```

Comment:
I guess, this meant to be removed? Or uncomment as it might be handy overload.

Reply:
It was unused, so I was going to remove it.   I guess I forgot - though it could be handy...